Documentation/design-docs/DotNetCore-SharedPackageStore.md

```diff
@@ -117,14 +106,16 @@ The host will probe in the following order of precedence for `dotnet run` and ap
 
 + `$CORE_SERVICING` on Unix or `%ProgramFiles(x86)%\coreservicing` on Windows.
 + `DOTNET_SHARED_STORE` in the chained order
-+ The user local shared package store in `%USERPROFILE%\.dotnet\store` or `$HOME/.dotnet/store`
++ The dotnet.exe relative shared package store
 + The global shared package store
 + The Shared Framework directory
 + ~~`DOTNET_HOSTING_OPTIMIZATION_CACHE` deprecated in favor of `DOTNET_SHARED_STORE`~~
 + `--additionalprobingpaths` specified in the command line
 + `runtimeOptions.additionalProbingPaths` (includes NuGet cache probe specified by the CLI for `dotnet run`)
 + Application `bin` directory
 
+**NOTE:**  `--additionalprobingpaths` can be passed template paths like below and the host will interpret `{arch}/{tfm}` appropriately to look for assets: 
+           `%USERPROFILE%\.dotnet\store\{arch}\{tfm}` or `$HOME/.dotnet/store/{arch}/{tfm}`
```

Comment:
I would suggest to give an example that does not tie in "store" so that it appears as generic as possible.